### ISSUE TITLE
【美食作家王刚】白灼血蛤 - https://youtu.be/oQuladip2S4

### ISSUE THREAD

**Issue by SilvesterDong** [immoonancient/YTSubtitles#1032]
**视频链接**：https://youtu.be/oQuladip2S4

视频时长7:06，预计发布时间为北京时间5月15日晚
这是刚哥最新的视频私享链接，方便大家提前添加字幕，文字稿在下面
文字稿可能会和视频有微小出入，请大家以视频为主。由于刚哥的合同问题，请大家务必不要泄露或分享刚哥未在YouTube发布的内容，包括文字，视频链接等，望大家理解！

# 请翻译以下全部内容

```
标题（翻译主要意思即可，不要超过 100 个字符）
厨师长分享：“白灼血蛤”的家常做法，10秒出锅，味道鲜嫩多汁

简介
大家好，朋友送了我一盒血蛤，由于这种蛤在生的时候肉现鲜红色，所以得名“血蛤”。做法非常简单，只需要将其洗净之后加入开水中烫5-15秒（根据自己的口味和对生熟的接受度来定），就可以出锅上盘。一般来说5秒基本属于全生，蛤需要撬开食用，里面有大量非常鲜的“血水”，但同样非常的腥。15秒的话基本属于全熟，蛤已经打开，虽说肉一样很嫩，但汁水比较少。所以推荐大家在家可以控制在10秒至12秒左右，个人认为对于没怎么吃过的朋友非常适合。

字幕
0:00:00.475,0:00:01.200
哈喽 大家好

0:00:01.400,0:00:06.720
最近我朋友从珠海给我寄了一盒这个血蛤

0:00:06.720,0:00:08.475
正好我伯爷没吃过了

0:00:08.650,0:00:12.212
然后今天就做一个什么呢

0:00:12.212,0:00:13.525
白灼血蛤来吃吧

0:00:13.800,0:00:17.100
这个雪蛤就是我刚刚开了一个给我四伯看

0:00:17.525,0:00:18.875
这个里面真的会流血

0:00:19.318,0:00:20.209
所以叫做雪蛤

0:00:20.524,0:00:24.134
而且这个特别适合血压高的人吃

0:00:24.466,0:00:26.293
然后它寒性非常大

0:00:26.293,0:00:27.884
如果肠胃不好的人

0:00:28.277,0:00:29.755
你尽量就不要吃这个

0:00:30.435,0:00:31.770
这个营养好的很四伯爷

0:00:32.233,0:00:33.901
看那个样子都可以

0:00:33.901,0:00:35.662
我去弄点水来洗哈都

0:00:36.570,0:00:37.775
把它倒进去洗一下

0:00:40.741,0:00:41.779
一起刷嘛

0:00:49.675,0:00:51.862
在这个血蛤在沿海地区

0:00:51.862,0:00:53.345
  有的朋友他喜欢吃生的

0:00:53.697,0:00:57.405
但是我们四川这边还是比较难接受这个

0:00:57.809,0:00:59.263
我四伯说生的他

0:01:00.398,0:01:01.931
生的咋个吃嘛

0:01:02.558,0:01:03.036
就是啊

0:01:04.161,0:01:05.535
还有一个就是要肠胃够好

0:01:07.710,0:01:09.485
肠胃不好的不可以吃

0:01:09.485,0:01:11.152
肠胃不好的吃了这个要

0:01:11.298,0:01:12.191
要拉肚子是不

0:01:12.191,0:01:13.269
嗯 肚子要遭

0:01:25.248,0:01:26.897
我伯爷说这个要多洗几下

0:01:26.897,0:01:29.837
他说有一点点泥的话都不放心吃起来

0:01:34.158,0:01:36.125
看嘛 洗完水都是清亮（干净）的

0:01:36.693,0:01:38.701
 我们这里可能有三四斤吧

0:01:38.924,0:01:40.770
我们刷了差不多一个小时

0:01:41.237,0:01:43.103
这个如果你们要吃的话

0:01:43.569,0:01:45.922
可以在海鲜市场买他们刷好的

0:01:45.922,0:01:48.782
不要像我这个因为是朋友送的嘛

0:01:48.782,0:01:51.026
我不可能叫他刷好了再给我送过来

0:01:51.805,0:01:55.841
我再切一点这个姜跟葱

0:01:56.125,0:01:59.716
然后再弄一点这个大蒜 还有辣椒

0:01:59.878,0:02:00.709
做一个蘸水

0:02:01.298,0:02:04.077
等下万一吃不惯这个纯味的

0:02:04.077,0:02:06.653
我们就蘸一个川味的

0:02:16.267,0:02:17.748
好了 这个就切好了

0:02:20.283,0:02:21.947
我们再切一点姜和葱

0:02:22.312,0:02:25.152
就是在煮它的时候放进去

0:02:25.172,0:02:28.843
真的这个东西吃的来的吃生的营养价值很好

0:02:28.843,0:02:29.452
是

0:02:30.608,0:02:33.367
就是很多可能都吃不来

0:02:33.367,0:02:34.381
我再放点酱油进去

0:02:36.146,0:02:37.018
诶呀 可以了

0:02:37.728,0:02:39.675
走嘛 帮我端哈四伯爷

0:02:42.540,0:02:44.397
这个时候咋个做

0:02:44.462,0:02:46.221
这个水开了放下去

0:02:46.221,0:02:47.947
煮十多秒钟

0:02:47.947,0:02:49.511
12秒钟就差不多

0:02:53.047,0:02:56.518
放入姜葱和水一起烧开

0:03:06.223,0:03:09.080
水开后放入血蛤   

0:03:16.358,0:03:19.630
12秒之后把血蛤捞出来

0:03:29.703,0:03:33.093
 这个打开里面还有血 看嘛这种最好吃

0:03:34.881,0:03:35.646
嗯

0:03:39.656,0:03:40.897
嗯 很鲜

0:03:42.295,0:03:43.799
我就是看到这肠肠多

0:03:43.799,0:03:46.121
不不不 不弄肠子 直接吃四伯爷

0:03:51.434,0:03:52.750
好了 我们来尝一下这个

0:03:53.358,0:03:53.652
怎么样

0:03:56.365,0:03:56.872
吃得惯不

0:03:57.462,0:03:57.895
吃得惯

0:03:57.895,0:04:00.074
你蘸一哈这个辣的嘛 你如果吃起腥的话

0:04:00.559,0:04:00.865
吃得来

0:04:01.181,0:04:01.746
安

0:04:02.328,0:04:02.983
吃得来

0:04:03.547,0:04:04.184
吃得来是不

0:04:04.184,0:04:04.548
嗯

0:04:06.131,0:04:08.196
来嘛尝哈这个蘸哈辣的怎么样

0:04:08.898,0:04:10.135
我试哈看嘛

0:04:11.036,0:04:11.664
嗯

0:04:12.701,0:04:15.285
蘸哈辣的更辣 吃起更酸辣

0:04:18.197,0:04:22.401
我们这个血蛤 有的人他吃这个血蛤

0:04:22.710,0:04:25.185
这水开了之后放进去十秒钟捞出来

0:04:25.622,0:04:26.404
要什么效果呢

0:04:26.404,0:04:29.625
就是要这种没有张开 自己把它掰开的这种

0:04:30.026,0:04:33.301
如果掰不开的话用硬币在后面拧一下就开了

0:04:34.066,0:04:37.578
这个打开了之后一定要见到有红色的液体

0:04:38.160,0:04:39.580
就是类似这种红色的

0:04:41.163,0:04:42.291
这个水特别鲜

0:04:48.552,0:04:51.261
很多人吃不惯这个（生的）

0:04:51.267,0:04:52.079
我还吃得来这个东西

0:04:52.319,0:04:52.936
吃得来是不

0:04:52.982,0:04:53.519
吃得来

0:04:55.702,0:04:57.942
先我还以为像鱼一样没去肠子这些

0:04:57.942,0:05:01.690
我还有点夹口 我还不敢吃

0:05:02.433,0:05:06.010
唉呀 我管三七二十一哦 我大到胆子吃2个看

0:05:06.113,0:05:09.142
哎 我吃起还好吃 安逸

0:05:09.222,0:05:09.564
是不

0:05:09.999,0:05:10.342
是呀

0:05:10.753,0:05:12.833
这个适合很多人吃

0:05:13.210,0:05:16.906
然后如果你肠胃比较寒的话

0:05:16.913,0:05:17.724
劝你就不要吃这个

0:05:17.724,0:05:19.278
因为你吃了这个很容易拉肚子

0:05:19.633,0:05:21.256
这个是很有凉性嘎

0:05:21.347,0:05:21.862
嗯

0:05:21.862,0:05:24.170
还有就是在女性生理期的时候不要吃这个

0:05:24.639,0:05:26.616
再一个就是产后不要吃这个

0:05:27.153,0:05:28.547
这个东西特别适合什么（人）呢

0:05:29.243,0:05:30.123
适合老年人吃

0:05:30.593,0:05:31.804
然后适合孕妇

0:05:32.353,0:05:35.016
适合这个做完月子之后吃

0:05:36.067,0:05:37.347
营养很丰富

0:05:38.136,0:05:39.324
你整的啥子哦

0:05:39.896,0:05:41.007
我没吃过哟

0:05:41.790,0:05:42.454
血蛤

0:05:42.648,0:05:42.902
嗯

0:05:42.902,0:05:44.163
这个名字还多好听

0:05:44.544,0:05:46.783
伯娘你吃不来的话就蘸哈这个海椒

0:05:49.439,0:05:50.283
好吃

0:05:52.812,0:05:54.693
我们吃这个的时候把肉吃下去

0:05:54.723,0:05:55.962
我专门吃这种跑出来的

0:05:55.962,0:05:57.193
里面会有一点汤

0:05:59.126,0:06:00.275
这个汤特别的鲜

0:06:01.275,0:06:01.820
嗯 好吃

0:06:04.163,0:06:05.424
你高血压多吃点

0:06:06.171,0:06:08.775
高血脂 高胆固醇的人特别适合

0:06:08.872,0:06:09.507
这样子啊

0:06:09.551,0:06:09.977
嗯

0:06:11.193,0:06:13.604
我们这个东西的话在四川没有

0:06:13.663,0:06:15.171
因为它是在沿海地区才有的

0:06:15.581,0:06:16.887
所以你要吃的话

0:06:17.715,0:06:21.902
要么在网上订 要不就是托在海边的朋友

0:06:22.376,0:06:23.282
好吃真的好吃

0:06:23.318,0:06:24.078
寄过来

0:06:25.551,0:06:29.799
好啦 大家可以不是海边的朋友可以尝试一下

0:06:30.161,0:06:32.503
下次你哥哥回来 我喊他从珠海带点回来

0:06:32.696,0:06:33.167
嗯

0:06:35.195,0:06:35.629
好

0:06:35.629,0:06:36.969
那边他肯定买得到撒

0:06:36.969,0:06:37.572
 买得到

0:06:38.019,0:06:39.359
好了 本期视频就到这里

0:06:39.383,0:06:42.533
我们要吃这个美味的海鲜

0:06:43.716,0:06:44.440
血蛤

0:06:49.908,0:06:50.813
我还吃得来

0:06:50.861,0:06:51.393
好了 

0:06:51.417,0:06:52.913
本期视频就到这里 拜拜

0:06:53.553,0:06:55.038
做拜拜撒你

0:06:55.327,0:06:57.379
我还要吃啥子拜拜哦

0:06:58.828,0:07:00.385
我要吃舒服了才拜拜


```

**Comment by Yingzhang1122** [comment]
#标题（翻译主要意思即可，不要超过 100 个字符）
#厨师长分享：“白灼血蛤”的家常做法，10秒出锅，味道鲜嫩多汁
Chef Wang shares: Home-style 'Bai Zhuo Xue Ha' (blanched blood clam). 10 seconds of cooking, juicy, tender, and yummy!

#简介
#大家好，朋友送了我一盒血蛤，由于这种蛤在生的时候肉现鲜红色，所以得名“血蛤”。做法非常简单，只需要将其洗净之后加入开水中烫5-15秒（根据自己的口味和对生熟的接受度来定），就可以出锅上盘。一般来说5秒基本属于全生，蛤需要撬开食用，里面有大量非常鲜的“血水”，但同样非常的腥。15秒的话基本属于全熟，蛤已经打开，虽说肉一样很嫩，但汁水比较少。所以推荐大家在家可以控制在10秒至12秒左右，个人认为对于没怎么吃过的朋友非常适合。
Hi everyone, my friend sent me a box of blood clams, which is named for its blood-like color. It's very simple to cook blood clam - wash thoroughly and blanch in boiling water for 5 - 15 seconds (depending on your preference of flavor), and it's ready to serve! Generally, the clam boiled for 5 seconds is still raw and thus needs to be opened to eat. At this time, most of the tasty juice is well preserved, but the clam may be very fishy. If boiled for 15 seconds, the clam is well cooked and opens itself. Though the meat is still tender, more juice is lost. Therefore, I recommend a cooking time between 10 and 12 seconds, which is very suitable for those who have never tasted it.

#字幕
0:00:00.475,0:00:01.200
#哈喽 大家好
Hi everyone

0:00:01.400,0:00:06.720
#最近我朋友从珠海给我寄了一盒这个血蛤
Recently my friend sent me this box of blood clams from Zhuhai

0:00:06.720,0:00:08.475
#正好我伯爷没吃过了
My uncle has never tasted it before

0:00:08.650,0:00:12.212
#然后今天就做一个什么呢
So today we shall cook a ...

0:00:12.212,0:00:13.525
#白灼血蛤来吃吧
'Bai Zhuo Xue Ha' (blanched blood clam)

0:00:13.800,0:00:17.100
#这个雪蛤就是我刚刚开了一个给我四伯看
This clam... I just opened one to show my uncle

0:00:17.525,0:00:18.875
#这个里面真的会流血
That it surely 'bleeds'

0:00:19.318,0:00:20.209
#所以叫做雪蛤
So it's called 'blood clam'

0:00:20.524,0:00:24.134
#而且这个特别适合血压高的人吃
This clam is very suitable for those with hypertension

0:00:24.466,0:00:26.293
#然后它寒性非常大
But it is a type of 'cold food' (according to the traditional Chinese theory of food)

0:00:26.293,0:00:27.884
#如果肠胃不好的人
If your digestive system is not working well

0:00:28.277,0:00:29.755
#你尽量就不要吃这个
Please avoid eating this clam

0:00:30.435,0:00:31.770
#这个营养好的很四伯爷
This clam contains a lot of nutrients, uncle

0:00:32.233,0:00:33.901
#看那个样子都可以
It looks like so

0:00:33.901,0:00:35.662
#我去弄点水来洗哈都
Let me get some water to wash them

0:00:36.570,0:00:37.775
#把它倒进去洗一下
Pour them into the bowl and wash them

0:00:40.741,0:00:41.779
#一起刷嘛
Let's brush them together

0:00:49.675,0:00:51.862
#在这个血蛤在沿海地区
The blood clam... in coastal areas

0:00:51.862,0:00:53.345
#有的朋友他喜欢吃生的
Some people prefer eating blood clams

0:00:53.697,0:00:57.405
#但是我们四川这边还是比较难接受这个
But here in Sichuan, most people may not like raw clams

0:00:57.809,0:00:59.263
#我四伯说生的他
My uncle says that raw food...

0:01:00.398,0:01:01.931
#生的咋个吃嘛
Why do I eat raw clams? # 感觉这里翻译成how can I eat 有点生硬

0:01:02.558,0:01:03.036
#就是啊
That's true

0:01:04.161,0:01:05.535
#还有一个就是要肠胃够好
Another issue is that, you need to have a strong digestive system

0:01:07.710,0:01:09.485
#肠胃不好的不可以吃
Otherwise, don't eat this

0:01:09.485,0:01:11.152
#肠胃不好的吃了这个要
If your digestive system is not good enough, you'll...

0:01:11.298,0:01:12.191
#要拉肚子是不
Have diarrhea?

0:01:12.191,0:01:13.269
#嗯 肚子要遭
Yep. You'll have digestive problems

0:01:25.248,0:01:26.897
#我伯爷说这个要多洗几下
My uncle says that this should be washed for more times

0:01:26.897,0:01:29.837
#他说有一点点泥的话都不放心吃起来
He won't rest assured even if there's just a little bit mud left

0:01:34.158,0:01:36.125
#看嘛 洗完水都是清亮（干净）的
See? The water after washing is still clear

0:01:36.693,0:01:38.701
#我们这里可能有三四斤吧
We may have 3 to 4 Jin (~ 1.5-2 kg) of clams here

0:01:38.924,0:01:40.770
#我们刷了差不多一个小时
We've been brushing them for almost an hour

0:01:41.237,0:01:43.103
#这个如果你们要吃的话
If you also want to eat this

0:01:43.569,0:01:45.922
#可以在海鲜市场买他们刷好的
You can buy brushed ones at the seafood market

0:01:45.922,0:01:48.782
#不要像我这个因为是朋友送的嘛
Well since these are given by my friend

0:01:48.782,0:01:51.026
#不可能叫他刷好了再给我送过来
I can't have him brush these for me

0:01:51.805,0:01:55.841
#我再切一点这个姜跟葱
Let me cut some ginger and spring onion

0:01:56.125,0:01:59.716
#然后再弄一点这个大蒜 还有辣椒
Some garlic and chili pepper as well

0:01:59.878,0:02:00.709
#做一个蘸水
to make a dipping sauce

0:02:01.298,0:02:04.077
#等下万一吃不惯这个纯味的
If we are not used to the 'original' blanched clams

0:02:04.077,0:02:06.653
#我们就蘸一个川味的
We can enjoy them in Sichuan style

0:02:16.267,0:02:17.748
#好了 这个就切好了
Ok. This is done

0:02:20.283,0:02:21.947
#我们再切一点姜和葱
Let's cut some ginger and spring onion

0:02:22.312,0:02:25.152
#就是在煮它的时候放进去
Add them into the wok while boiling the clams

0:02:25.172,0:02:28.843
#真的这个东西吃的来的吃生的营养价值很好
For sure, the raw clams might be really good for your health

0:02:28.843,0:02:29.452
#是
Yep

0:02:30.608,0:02:33.367
#就是很多可能都吃不来
But they may not be acceptable to most people

0:02:33.367,0:02:34.381
#我再放点酱油进去
Let me add some light soy sauce

0:02:36.146,0:02:37.018
#诶呀 可以了
Ok. This is done!

0:02:37.728,0:02:39.675
#走嘛 帮我端哈四伯爷
Let's go. Would you help me with this bowl?

0:02:42.540,0:02:44.397
#这个时候咋个做
How do we cook it?

0:02:44.462,0:02:46.221
#这个水开了放下去
When the water is boiling, put the clams into the wok

0:02:46.221,0:02:47.947
#煮十多秒钟
Boil them for slightly more than 10 seconds

0:02:47.947,0:02:49.511
#12秒钟就差不多
12 seconds might be ok

0:02:53.047,0:02:56.518
#放入姜葱和水一起烧开
Add ginger and spring onion into the wok and boil them with the water

0:03:06.223,0:03:09.080
#水开后放入血蛤   
When the water is boiling, put the blood clams into the wok

0:03:16.358,0:03:19.630
#12秒之后把血蛤捞出来
After 12 seconds, take the clams out of the water

0:03:29.703,0:03:33.093
#这个打开里面还有血 看嘛这种最好吃
This one still has 'blood'. See? This type is the most delicious

0:03:34.881,0:03:35.646
#嗯
Yep

0:03:39.656,0:03:40.897
#嗯 很鲜
Hmm. Very tasty

0:03:42.295,0:03:43.799
#我就是看到这肠肠多
I see a lot of intestines...

0:03:43.799,0:03:46.121
#不不不 不弄肠子 直接吃四伯爷
No no no.. don't remove the intestines. Just eat the whole thing, uncle

0:03:51.434,0:03:52.750
#好了 我们来尝一下这个
Ok. Let's try the dipping sauce

0:03:53.358,0:03:53.652
#怎么样
How do you like it?

0:03:56.365,0:03:56.872
#吃得惯不
Can you get used to it?

0:03:57.462,0:03:57.895
#吃得惯
Yes

0:03:57.895,0:04:00.074
#你蘸一哈这个辣的嘛 你如果吃起腥的话
Just dip the clam in the spicy sauce here, if you think it's too fishy

0:04:00.559,0:04:00.865
#吃得来
I think it's ok

0:04:01.181,0:04:01.746
#安
Huh?

0:04:02.328,0:04:02.983
#吃得来
I think it's ok

0:04:03.547,0:04:04.184
#吃得来是不
You like it?

0:04:04.184,0:04:04.548
#嗯
Yep

0:04:06.131,0:04:08.196
#来嘛尝哈这个蘸哈辣的怎么样
Come and try how it's like with this spicy dipping sauce

0:04:08.898,0:04:10.135
#我试哈看嘛
Let me try

0:04:11.036,0:04:11.664
#嗯
Wow

0:04:12.701,0:04:15.285
#蘸哈辣的更辣 吃起更酸辣
It tastes spicier with this sauce. Sour and spicy

0:04:18.197,0:04:22.401
#我们这个血蛤 有的人他吃这个血蛤
This blood clam... Some people like eating it...

0:04:22.710,0:04:25.185
#这水开了之后放进去十秒钟捞出来
after just 10 seconds of boiling

0:04:25.622,0:04:26.404
#要什么效果呢
They like the status where...

0:04:26.404,0:04:29.625
#就是要这种没有张开 自己把它掰开的这种
the clams are not opened yet. You still need to open them

0:04:30.026,0:04:33.301
#如果掰不开的话用硬币在后面拧一下就开了
If you can't open the clam with your hands, you can use a coin to open it from the back

0:04:34.066,0:04:37.578
#这个打开了之后一定要见到有红色的液体
You should see the red liquid after opening

0:04:38.160,0:04:39.580
#就是类似这种红色的
Just like this... the red liquid

0:04:41.163,0:04:42.291
#这个水特别鲜
The 'juicy' is extremely tasty and umami

0:04:48.552,0:04:51.261
#很多人吃不惯这个（生的）
Many people are not used to this (raw clam)

0:04:51.267,0:04:52.079
#我还吃得来这个东西
Well, I'm ok with it

0:04:52.319,0:04:52.936
#吃得来是不
You like it?

0:04:52.982,0:04:53.519
#吃得来
Yup

0:04:55.702,0:04:57.942
#先我还以为像鱼一样没去肠子这些
Previously I thought it's like a fish with intestines...

0:04:57.942,0:05:01.690
#我还有点夹口 我还不敢吃
I thought it might taste weird so I was reluctant #夹口貌似是指食物中含碱过多或者没熟而口感不佳，使舌头感到发麻、干涩，这里就意译了

0:05:02.433,0:05:06.010
#唉呀 我管三七二十一哦 我大到胆子吃2个看
Then I was like 'screw it! Just be bold and eat some'

0:05:06.113,0:05:09.142
#哎 我吃起还好吃 安逸
Surprisingly this is very delicious. Nice!

0:05:09.222,0:05:09.564
#是不
Isn't it?

0:05:09.999,0:05:10.342
#是呀
Tasty indeed

0:05:10.753,0:05:12.833
#这个适合很多人吃
The blood clam is suitable for many people

0:05:13.210,0:05:16.906
#然后如果你肠胃比较寒的话
However if you have a relatively poor digestive system

0:05:16.913,0:05:17.724
#劝你就不要吃这个
I would not recommend this to you

0:05:17.724,0:05:19.278
#因为你吃了这个很容易拉肚子
Because you'll easily have diarrhea after eating them

0:05:19.633,0:05:21.256
#这个是很有凉性嘎
It's pretty 'cold', right?

0:05:21.347,0:05:21.862
#嗯
Yes

0:05:21.862,0:05:24.170
#还有就是在女性生理期的时候不要吃这个
In addition, for females during periods, please don't eat this

0:05:24.639,0:05:26.616
#再一个就是产后不要吃这个
Also, females who just give birth should avoid eating this

0:05:27.153,0:05:28.547
#这个东西特别适合什么（人）呢
The blood clam is suitable for these populations...

0:05:29.243,0:05:30.123
#适合老年人吃
Such as elderly people

0:05:30.593,0:05:31.804
#然后适合孕妇
Pregnant woman

0:05:32.353,0:05:35.016
#适合这个做完月子之后吃
As well as females after the 'postpartum confinement' #感觉这个表达本身是带有对'坐月子'的反对，虽然我也反对坐月子...如果中性一点的翻译方法可以用postpartum period

0:05:36.067,0:05:37.347
#营养很丰富
As it contains rich nutrients

0:05:38.136,0:05:39.324
#你整的啥子哦
What do we have here?

0:05:39.896,0:05:41.007
#我没吃过哟
I've never had this before

0:05:41.790,0:05:42.454
#血蛤
Blood clams

0:05:42.648,0:05:42.902
#嗯
Hmm

0:05:42.902,0:05:44.163
#这个名字还多好听
Interesting name

0:05:44.544,0:05:46.783
#伯娘你吃不来的话就蘸哈这个海椒
Aunt, if you can't get used to it, you can dip it in the chili pepper sauce

0:05:49.439,0:05:50.283
#好吃
Delicious!

0:05:52.812,0:05:54.693
#我们吃这个的时候把肉吃下去
When eating this, we eat the meat first

0:05:54.723,0:05:55.962
#我专门吃这种跑出来的
I'll specially pick the ones out of the shell

0:05:55.962,0:05:57.193
#里面会有一点汤
There's a little bit 'juice' in the shell

0:05:59.126,0:06:00.275
#这个汤特别的鲜
which is very tasty and umami

0:06:01.275,0:06:01.820
#嗯 好吃
Hmm. Yummy!

0:06:04.163,0:06:05.424
#你高血压多吃点
You have hypertension. Just eat more

0:06:06.171,0:06:08.775
#高血脂 高胆固醇的人特别适合
The clam is very suitable for those with high blood fat and cholesterol levels

0:06:08.872,0:06:09.507
#这样子啊
Alright

0:06:09.551,0:06:09.977
#嗯
Yep

0:06:11.193,0:06:13.604
#我们这个东西的话在四川没有
We don't have this in Sichuan province

0:06:13.663,0:06:15.171
#因为它是在沿海地区才有的
Because it can only be harvested in coastal areas

0:06:15.581,0:06:16.887
#所以你要吃的话
So if you would like to have a try

0:06:17.715,0:06:21.902
#要么在网上订 要不就是托在海边的朋友
You can either order some on the internet, or ask your friends by the sea...

0:06:22.376,0:06:23.282
#好吃真的好吃
Delicious. Indeed.

0:06:23.318,0:06:24.078
#寄过来
... to mail some to you

0:06:25.551,0:06:29.799
#好啦 大家可以不是海边的朋友可以尝试一下
Ok. I recommend everyone... who doesn't live by the sea to try this

0:06:30.161,0:06:32.503
#下次你哥哥回来 我喊他从珠海带点回来
Next time when your cousin returns from Zhuhai, I'll ask him to bring some back #这里哥哥大概率指的是堂哥？

0:06:32.696,0:06:33.167
#嗯
Okay

0:06:35.195,0:06:35.629
#好
Ok

0:06:35.629,0:06:36.969
#那边他肯定买得到撒
He can definitely buy some there, right?

0:06:36.969,0:06:37.572
#买得到
Yes of course

0:06:38.019,0:06:39.359
#好了 本期视频就到这里
Ok. That's the end of this video

0:06:39.383,0:06:42.533
#我们要吃这个美味的海鲜
We are gonna enjoy the delicious seafood

0:06:43.716,0:06:44.440
#血蛤
Blood clam

0:06:49.908,0:06:50.813
#我还吃得来
I like it

0:06:50.861,0:06:51.393
#好了
Ok

0:06:51.417,0:06:52.913
#本期视频就到这里 拜拜
That's the end of this video. Bye-bye!

0:06:53.553,0:06:55.038
#做拜拜撒你
Aunt: Say bye-bye

0:06:55.327,0:06:57.379
#我还要吃啥子拜拜哦
I'm still eating. Why bye-bye?

0:06:58.828,0:07:00.385
#我要吃舒服了才拜拜
I'll say bye-bye only after I'm done
